examples/chef/common/chef-laundry-washer-mode.cpp

```diff
@@ -25,6 +25,7 @@ template <typename T>
 using List              = chip::app::DataModel::List<T>;
 using ModeTagStructType = chip::app::Clusters::detail::Structs::ModeTagStruct::Type;
 
+#ifdef MATTER_DM_PLUGIN_LAUNDRY_WASHER_MODE_SERVER
```

Comment:
Can this optionality be better controlled by adding/removing the file to the build?
If such an ifdef to include / exclude code is the precedent, could this be moved to the top of the file to make it clear that it applies to the entire file and not parial functionality?